extensions/typescript-language-features/src/languageFeatures/renameOnType.ts

```diff
@@ -0,0 +1,69 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { ClientCapability, ITypeScriptServiceClient } from '../typescriptService';
+import { conditionalRegistration, requireSomeCapability } from '../utils/dependentRegistration';
+import { DocumentSelector } from '../utils/documentSelector';
+import * as typeConverters from '../utils/typeConverters';
+
+
+class TypeScriptOnTypeRenameProvider implements vscode.OnTypeRenameProvider {
+
+	private static enabledKinds = new Set<string>([
+		'let', 'const', 'local var', 'parameter'
```

Comment:
[nit] Use the [`protocol.const` static variables](https://github.com/microsoft/vscode/blob/9d6054c99d63b934148051f1fb5ae22e926cc169/extensions/typescript-language-features/src/protocol.const.ts#L6) here (this lets us use find all references on them)

Reply:
check